frontend/src/components/Question.tsx

```diff
@@ -8,8 +8,20 @@ import {
   Stack,
   Typography,
 } from "@mui/material";
+import DOMPurify from "dompurify";
+import parse from "html-react-parser";
 
 export const Question = () => {
+  /* Question Content */
+  const title = "Two Sum";
+  const content =
```

Comment:
Could we add better stylings for the \<code> and \<pre> tags? Mainly just a grey background would make it more noticeable as code blocks.